docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,52 @@
-That's why submitting transactions to the priority queue is no longer free.
-Users need to pay a certain fee to the operator for processing their transactions. It is really hard to calculate the accurate fee in a permissionless way.
-Thus, the fee for a transaction is equal to `txBaseCost * gasPrice`. The `gasPrice` is the gas price of the users' transaction, while `txBaseCost` is the base cost for the transaction, which depends on its parameters (e.g. `gas_limit` for `Execute` transaction).
+- Find out [how to estimate gas](../../how-to/estimate-gas.md) for different scenarios.
+- Find out [how to send a transaction from L1 to L2](../../how-to/send-transaction-l1-l2.md) with zkSync Era.
 
-Thirdly, the operator can not commit to processing every transaction within `X` days. Again, this is needed to prevent spam attacks on the priority queue. We changed this rule to the following one:
+## L2 to L1 
 
-- The operator must do at least `X` amount of work (see below) on the priority queue or the priority queue should be empty.
+L2 to L1 communication is based on transferring the data as a message, and not on L1 transaction execution. 
 
-In other words, we require the operator to do its best instead of requiring a strict deadline. The measure of "the work" is still to be developed. Most likely it will be the number of `gas` the priority operations used.
+- Find out [how to send a message from L2 to L1](../../how-to/send-message-l2-l1.md) with zkSync Era.
 
-In the future, we will also add the ability to "prioritize" L1->L2 transactions, allowing users to speed up the inclusion of their transactions in exchange for paying a higher fee to the operator.
-
-## Priority mode
-
-If the operator fails to process the needed L1 transactions, the system enters in 'Priority mode'. In this mode, everyone can become an operator by staking tokens. The exact details of the priority mode are still under development and will be described in more detail closer to the mainnet launch.
-
-To reduce risks, the alpha mainnet will start with a mechanism to instantly stop and upgrade the network, which contradicts the purpose of the priority mode. Priority mode will be gradually introduced in the following releases.
-
-## L2 -> L1 messaging
-
-The [L2 -> L1 communication](./l2-l1.md), in contrast to L1 -> L2 communication, is based only on transferring the information, and not on the transaction execution on L1. It is a built-in feature, which is made up of two parts: sending a message from L2 and reading it on L1. The first is implemented as a call to an L2 system smart contract. And the second is implemented on the zkSync L1 smart contract as a getter function.
+## Priority queue
 
-### Sending messages
+zkSync Era supports general smart contract computation which includes complex operations. For this reason, some principles had to evolve in order to preserve the stability of the network.
```

Comment:
I think that when we use `evolve` we need to show compared to what the evolution is done, I guess a lot of people that make come to version 2 have on prior experience with version 1

Reply:
removed